canaries/README.md

```diff
@@ -0,0 +1,8 @@
+Most used commands
+------------------
+
+# Generate signature
+
+```
+gpg --default-key <my_key> --armor --output foo.txt.sig.<id> --sign --detach-sign foo.txt
```

Comment:
@pietrushnic the GPG command was wrong. Already fixed it.

- `-k` option lists available keys, not for use with sign or detach sign
- fixed typo in `--detach-sign`
- key should be passed with `--default-key` for example
- from GPG man: `Syntax: gpg [options] [files]` so options and commands go first, the last one should be file to sign

Just FYI.

Reply:
@miczyg1 sorry I should check it first, I used `-k` since you have to be sure what key you choose, but apparently, I did that incorrectly.